biome.json

```diff
@@ -25,7 +25,8 @@
       "**/*.d.ts",
       ".codesandbox/ci.json",
       "build.icon-list.js",
-      "**/*.hbs"
+      "**/*.hbs",
+      "docs"
```

Comment:
praise: good catch. debugged this separately in nora's branch too. Sorry for the miss here